config.go

```diff
@@ -290,6 +322,24 @@ func DefaultWANConfig() *Config {
 	return conf
 }
 
+// IPMustBeChecked return true if IPAllowed must be called
+func (c *Config) IPMustBeChecked() bool {
+	return c.CidrsAllowed != nil
```

Comment:
```suggestion
	return len(c.CidrsAllowed) > 0
```
Generally every time I treat a `nil` slice differently than an empty slice I ultimately come to regret it. As it doesn't make much sense to have en empty cidr allow list mean "nothing is allowed" since that would mean memberlist couldn't function, it should be semantically fine to just check for emptyness instead of nilness.